views/includes/page-templates/managing-officer-corporate.html

```diff
@@ -75,8 +75,12 @@ <h2 class="govuk-heading-l">Who can we contact about this managing officer?</h2>
         name: "contact_email",
         value: contact_email
       }) }}
-
-      {% include "includes/inset-text.html" %}
+      
+      {{ govukInsetText({
```

Comment:
Jut noticed that in your other PR for update remove (UAR01582) that you have a new file for this section:` insert-text-cmo.html `that could be used here.